src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -165,6 +171,30 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 		}.store(in: &subscriptions)
 	}
 
+	func forcedSubmitData(completion: @escaping (Result<Void, PPASError>) -> Void) {
```

Comment:
There is some code duplication now.
Can we somehow reuse the code?

Reply:
ah yeah, I think we can do :) let me refactor this.